salt/config/__init__.py

```diff
@@ -1065,6 +1072,10 @@
     'event_match_type': 'startswith',
     'minion_restart_command': [],
     'pub_ret': True,
+    'proxy_host': None,
```

Comment:
Personally I dont agree but, I'm happy to make it so for consistency with other defaults. patched